.vscode/extensions.json

```diff
@@ -1,5 +1,6 @@
 {
   "recommendations": [
-    "denoland.vscode-deno"
+    "denoland.vscode-deno",
+    "stackbreak.comment-divider"
```

Comment:
I don't know if it's valid to add this.

Reply:
I can remove it, I used it only for the comment dividers.

Reply:
What do you think apart from that? Is it okay to rename the `setupGithooks` function to `setup`? The module is used as CLI anyways, right?